### ISSUE TITLE
extra binary files in SRC folder

### ISSUE THREAD

**Issue by dougransom** [dictation-toolbox/natlink#136]
there are some build targets in https://github.com/dictation-toolbox/natlink/tree/master/pythonsrc/src/natlink  that seem to have made their way into
https://github.com/dictation-toolbox/natlink/releases/tag/NATLINK_5_3_4

shouldn't hurt anything but how did they get there? Anyways they should be deleted.

**Comment by LexiconCode** [comment]
.gitignore should be updated to ignore extra binary files

**Comment by dougransom** [comment]
Yes, but I think they should not have been placed in that folder during the build process so we should investigate why they are in that folder.
